### ISSUE TITLE
Incorrect climatization events

### ISSUE THREAD

**Issue by amenychtas** [tillsteinbach/VWsFriend#387]
After the OTA 3.0 update I have noticed some incorrect climatization events after locking the car. See the picture below: 

![image](https://user-images.githubusercontent.com/3939897/211351144-4a238b47-dd43-4484-b644-a2686d8265db.png)

If I can be of help by providing debug messages, please let me know.

**Comment by amenychtas** [comment]
This seems strange because I didn't enable the climatization, neither changed any setting in the car or the app. And still these events are being recorded.

**Comment by tillsteinbach** [comment]
You don't have climatization after unlock enabled?

**Comment by amenychtas** [comment]
> You don't have climatization after unlock enabled?

No, I think I haven't used the climatization at all after the update (3 days ago). As far as I can see though, these climatization events come after locking the car (after trip ended). That's why it seems ackward to me.

Also the related feature is disabled in the app settings.

![image](https://user-images.githubusercontent.com/3939897/211358910-19885a58-8cf5-4db3-afed-327ef2fc5e57.png)

**Comment by tillsteinbach** [comment]
When you look in the Car Overview under states. Can you hover over the climatization and see how long the climatization is on? And it would be interesting what the WeConnect ID App shows at the same time.

**Comment by amenychtas** [comment]
Here is the status: 
![image](https://user-images.githubusercontent.com/3939897/211360069-43e924d8-5334-44fb-8a87-e0cf5917985e.png)

The WeConnect ID App only shows that Air Conditioning is Off.

**Comment by amenychtas** [comment]
Another event from this morning: 

![image](https://user-images.githubusercontent.com/3939897/211507524-e6dcb695-5fcd-4a3a-87ce-a8c5bf7cc800.png)

![image](https://user-images.githubusercontent.com/3939897/211507575-cdbfa72b-c693-4cf2-986b-21b99ec6818e.png)

**Comment by tillsteinbach** [comment]
is this still the case?

**Comment by amenychtas** [comment]
Unfortunately yes, and I cannot correlate it with something. It happens randomly after the update to v3.0. 

These are from yesterday:
![image](https://user-images.githubusercontent.com/3939897/215268198-f9e772ed-977e-43ca-ab9c-8230a46cc2bf.png)

In all cases, the duration of the ventilation is 10 seconds, and are recorded after `trip ended` event.
![image](https://user-images.githubusercontent.com/3939897/215268300-54475a3f-d380-4b45-880d-e9e13cddaab4.png)

**Comment by tillsteinbach** [comment]
Hmmm, this is really strange! I have not seen that before and no other user reported something similar yet.

**Comment by tillsteinbach** [comment]
I still cannot reproduce this
